2.0.1/buster/Dockerfile

```diff
@@ -108,7 +108,7 @@ RUN apt-get update \
 ARG VERSION="2.0.1-1.*"
 ARG SUBMODULES="async,azure,amazon,celery,elasticsearch,google,password,cncf.kubernetes,mysql,postgres,redis,slack,ssh,statsd,virtualenv"
 ARG AIRFLOW_MODULE="astronomer_certified[${SUBMODULES}]==$VERSION"
-ARG AIRFLOW_VERSION="2.0.1"
+ARG AIRFLOW_VERSION="2.0.0"
```

Comment:
Given that this is specifically for 2.0.1, what is this change to 2.0.0 doing here?

Reply:
Currently we do not have a constraints on 2.0.1. This is a patch release. So I assume it will be the same during the patch releases.

Reply:
But the file is literally called `ap-airflow/2.0.1/buster/Dockerfile`

Reply:
Yes this should be called 2.0.1

Reply:
```suggestion
ARG AIRFLOW_VERSION="2.0.1"
```